contrib/k8s/matchbox-ingress.yaml

```diff
@@ -15,7 +15,7 @@ spec:
 apiVersion: extensions/v1beta1
```

Comment:
While you're updating, mind changing these both to networking.k8s.io/v1beta1

Reply:
Not a problem - and done